jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -326,9 +335,7 @@
             job-env: |
                 export JENKINS_PUBLISHED_VERSION="ci/latest-1.3"
                 export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Ingress\]"
-                export PROJECT="kubernetes-ingress-1-2"
-                # TODO: Enable this when we've split 1.3 tests into another project.
-                export FAIL_ON_GCP_RESOURCE_LEAK="false"
```

Comment:
It shouldn't be necessary. I'm actually not sure it was necessary before, either - the 1.2 tests were completely removed, so there was nothing running in the same project. I manually grepped to verify there was nothing conflicting.

However, the presence of this option makes this commit safe to do atomically, rather than draining onto the new projects first and then adding the release-1.2 tests back.